### DISCUSSION TITLE
my progress s getting thrown back

### DISCUSSION THREAD

**Discussion by RemmyDev1** [codewars/codewars.com#2528] [General]
so i would finish a 1kyu kata then my progress would be sent back and say that my rank is 8kyu pls help if possible

**Comment by hobovsky** [comment]
Could you share your codewars username,  or a link to your profile?

**Comment by RemmyDev1** [comment]
its linked in my github koko1237

**Comment by hobovsky** [comment]
The account is recognized as a repeating offender of copying solutions and submitting them as own, to get point and progress rewards. Theres a ban request filed against it, and it has A LOT of invalidated solutions.

I do not think the account is worth saving. You'd better delete it, and if you still want to use Codewars,  create a new one and this time solve kata by yourself and play fair.
